bagit.py

```diff
@@ -94,53 +98,60 @@ def find_locale_dir():
 
 For more information or to contribute to bagit-python's development, please
 visit %(PROJECT_URL)s
-""") % globals()
+"""
+    )
+    % globals()
+)
 
 # standard bag-info.txt metadata
 STANDARD_BAG_INFO_HEADERS = [
-    'Source-Organization',
-    'Organization-Address',
-    'Contact-Name',
-    'Contact-Phone',
-    'Contact-Email',
-    'External-Description',
-    'External-Identifier',
-    'Bag-Size',
-    'Bag-Group-Identifier',
-    'Bag-Count',
-    'Internal-Sender-Identifier',
-    'Internal-Sender-Description',
-    'BagIt-Profile-Identifier',
+    "Source-Organization",
+    "Organization-Address",
+    "Contact-Name",
+    "Contact-Phone",
+    "Contact-Email",
+    "External-Description",
+    "External-Identifier",
+    "Bag-Size",
+    "Bag-Group-Identifier",
+    "Bag-Count",
+    "Internal-Sender-Identifier",
+    "Internal-Sender-Description",
+    "BagIt-Profile-Identifier",
     # Bagging-Date is autogenerated
     # Payload-Oxum is autogenerated
 ]
 
-# The getattr call can be replaced with direct access when we're willing to
-# break compatibility with Python 2.6:
-CHECKSUM_ALGOS = getattr(hashlib, 'algorithms_guaranteed', set(('md5', 'sha1', 'sha256', 'sha384', 'sha512')))
-DEFAULT_CHECKSUMS = ['sha256', 'sha512']
+CHECKSUM_ALGOS = hashlib.algorithms_guaranteed
```

Comment:
Unfortunately this changed line fails under Python 2.7.0-2.7.8 because `hashlib.algorithms_guaranteed` was introduced only in Python 2.7.9, see https://docs.python.org/2.7/library/hashlib.html#hashlib.algorithms_guaranteed
Can you please revert this change? I can open a PR if you prefer.

xref. https://github.com/fair-research/bdbag/commit/86517b9ba89524c3e1328eea7f4537552f0af82e

Reply:
Ouch, that’s unfortunate. Even we have upgraded to get the security updates. I don’t want to revert this since it’s forward-looking but perhaps we could replace it with a getattr call using a default which is just something like the core MD-5, SHA-1, SHA-{256,384,512} list most people use.